### ISSUE TITLE
Examples panicking after update to glutin 0.17

### ISSUE THREAD

**Issue by fungos** [Gekkio/imgui-rs#68]
[This commit](https://github.com/Gekkio/imgui-rs/commit/f33567a1dca83060f4535a9dc5c2e8ed8463edb9) caused imgui-rs test_window and test_window_impl to panic (on windows at least, haven't tried on linux).

It adds an unimplemented call to handle a resize event, this is what is causing the application to panic, see:

```
thread 'main' panicked at 'not yet implemented', examples\support\mod.rs:40:37
stack backtrace:
   0: std::sys_common::backtrace::_print
             at C:\projects\rust\src\libstd\sys_common\backtrace.rs:94
   1: std::panicking::default_hook::{{closure}}
             at C:\projects\rust\src\libstd\panicking.rs:379
   2: std::panicking::default_hook
             at C:\projects\rust\src\libstd\panicking.rs:396
   3: std::panicking::rust_panic_with_hook
             at C:\projects\rust\src\libstd\panicking.rs:610
   4: std::panicking::begin_panic<str*>
             at C:\projects\rust\src\libstd\panicking.rs:571
   5: test_window_impl::support::run::{{closure}}<closure>
             at .\examples\support\mod.rs:40
   6: winit::platform::platform::events_loop::EventsLoop::poll_events<closure>
             at C:\Users\Utilisateur\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.7.5\src\platform\windows\events_loop.rs:154
   7: winit::EventsLoop::poll_events<closure>
             at C:\Users\Utilisateur\.cargo\registry\src\github.com-1ecc6299db9ec823\winit-0.7.5\src\lib.rs:219
   8: test_window_impl::support::run<closure>
             at .\examples\support\mod.rs:33
   9: test_window_impl::main
             at .\examples\test_window_impl.rs:113
  10: panic_unwind::__rust_maybe_catch_panic
             at C:\projects\rust\src\libpanic_unwind\lib.rs:98
  11: std::rt::lang_start
             at C:\projects\rust\src\libstd\rt.rs:54
  12: main
  13: __scrt_common_main_seh
             at f:\dd\vctools\crt\vcstartup\src\startup\exe_common.inl:259
  14: BaseThreadInitThunk
error: process didn't exit successfully: `C:\Danny\imgui-rs\target\debug\examples\test_window_impl.exe` (exit code: 101)
```

**Comment by brendanzab** [comment]
My bad! I was keeping tot he gfx example's style, and we never did handle the resize event for the glium examples anyway. I added `unimplemented!()`, but never got around to figuring out how!

**Comment by Gekkio** [comment]
Oops, should've noticed this in the code review. I'll try to get it fixed tomorrow :sweat_smile:

**Comment by Gekkio** [comment]
This was fixed in 51329d5938b446d272f344ff1721711cfa265585
